pages/bookmarks.jsx

```diff
@@ -49,10 +49,10 @@ export default React.createClass({
     return(<p>Hi {user.username}! <button onClick={this.handleLogOutBtnClick} className="btn btn-link inline-link">Log out</button>.</p>);
   },
   getFailurePrompt() {
-    return (<p>Only Mozilla staff can login now as we test this new platform. Meanwhile, your favorited projects can still be remembered on this device.</p>);
+    return (<p>Something went wrong! Please refresh and try again.</p>);
```

Comment:
Is there any plan in the pipeline for sign up if they don't have an account(here Google). I added that message as we have a bug #454 but again if there is a provision to add sign up options in future we may add some buttons. But I still believe there should some sort message in case of a loading error.

@mmmavis @Pomax